app/res/xml/commcare_preferences.xml

```diff
@@ -32,7 +32,7 @@
         android:defaultValue="no"
         android:entries="@array/pref_enabled_labels"
         android:entryValues="@array/pref_enabled_vals"
-        android:key="cc-newest-version-from-hq"
+        android:key="cc-update-to-unstarred-builds"
```

Comment:
Hm, I didn't. I imagine the extent of the issue will be that if you already had it enabled, you will have to re-enable it, since the preference for which a "YES" value was saved is no longer the one we are using. I don't think I'm concerned by that since this setting is only ever used for debugging purposes (as opposed to being a setting that projects turn on permanently for all of their users and expect it to stay that way). In other words, it seems like OK behavior to me for an app builder to have to re-enable this particular setting after updating to a new version of CommCare. What do you think?